specs/Terminate Spec.md

```diff
@@ -17,42 +17,54 @@ PowerToys is mainly targeted towards Windows Power Users though it is available
   
 **1.3. Problem Statement and Supporting Customer Insights**
 
-Windows users need a quicker, more efficient way to completely shut down an app whenever it is unresponsive. The team is still required to find a proper name for this PowerToy.
+Windows users need an accessible mechanism to completely kill a process when it is being unresponsive and hindering work flow. The team is still required to find solutions for:
 
-**1.4. Existing Solutions or Expectations**
+- A publicly accpetable name for this PowerToy.
+- The degree to which the process will be terminated.
 
-Users can already acheive the function of this PowerToy by accessing clicking End Task through Task Manager on Windows. Users need to be running Windows 10 and have PowerToys installed to use it.
+**1.4. Existing Solutions or Expectations**
 
+The current methods a user can close a running program in Windows include:
+- Clicking on the close button in the program
+- Closing the program via task manager
+- Using the keyboard shortcut Alt + F4 to close the program
 
 **1.5. Goals/Non-Goals**
 
 - Develop this PowerToy and have sufficient time for testing and integration within our assigned 8 weeks for the project. 
 
-- Come up with a suitable name for this PowerToy, preferrably one accepted by the community.
+- Meet customers expectation with end result of project.
 
 ## 2. Definition of Success
 
 **2.1. Expected Impact: Customer, and Technology Outcomes, Experiments + Measures**
 
-There is currently a sizable number of people who are following the PowerToys project. We hope to meet these people's expectations and receive good feedback from the community when we are done with this PowerToy.
+The PowerToys repo currently has 200+ people watching, over 4000 stars and 109 forks on github despite having an empty repo. Also, this particular PowerToy received a rating of 3.44/5 in the survey we sent out to the community asking them to rate how useful they think it will be. After the PowerToys repo is made public, we hope to see at least a 5% increase in all the Github statistics aforementioned. On completion, a follow up survey will be sent out to the community and an increase in the rating to about 3.75/5 will be a good indicator that we met our customers satisfaction.
```

Comment:
@mscraigloewen I want to add more to this. Maybe some download and install stats too, but what do you think about what I have right now?